helmfile/values/harbor.yaml.gotmpl

```diff
@@ -110,6 +117,10 @@ registry:
   tolerations:  {{- toYaml .Values.harbor.tolerations | nindent 4 }}
 
 notary:
```

Comment:
seems to be another notary key starting with line 173.
I didn't manage to find if it's a duplicate or not

Reply:
Oooh, nice find! Seems to be a duplicate. Let me merge those two